ucsmsdk/ucsmo.py

```diff
@@ -107,6 +107,30 @@ def _set_mo_prop_value(self, kwargs):
     def _is_unknown_property(self, prop):
         return prop not in self.prop_meta
 
+    def __get_prop(self, name):
+        return object.__getattribute__(self, name)
+
+    def check_prop_match(self, **kwargs):
+        for prop_name in kwargs:
+            if prop_name not in self.prop_meta.keys():
+                raise ValueError("Invalid Property Name Exception - "
+                                 "Class [%s]: Prop <%s> "
+                                 % (self.__class__.__name__, prop_name))
+
+            if kwargs[prop_name] != self.__get_prop(prop_name):
```

Comment:
Can we use `getattr(self, prop_name)` directly here, since we are just fetching the value of the property and not over-riding the default behaviour ?